packages/perseus/src/widgets/interactive-graph.tsx

```diff
@@ -135,6 +135,24 @@ type DefaultProps = {
     graph: Props["graph"];
 };
 
+// TODO: there's another, very similar getSinusoidCoefficients function
+// they should probably be merged
+export function getSinusoidCoefficients(
```

Comment:
There are two `getSinusoidCoefficients` functions that are nearly identical except one returns an array and one returns an object. I didn't want to go down the refactoring rabbit-hole, so I kept both but made this an exported function rather than a static method.